templates/chats.html

```diff
@@ -6,9 +6,12 @@
     <div class="col-md-6">
       <h2>⌘ StartIT tērzētava</h2>
     </div>
-    <div class="col-md-6">
+    <div class="col-md-4">
       <p id="versija"></p>
     </div>
+    <div class="col-md-2">
+      <button id="mutebtn" value="on"></button>
```

Comment:
Nav jau īsit mutebtn, poga jau ieslēdz/izslēdz.

Reply:
Taisnība. Pārdēvēju.
Sākumā mēģināju izmantot metodi muted, bet tā šajā risinājumā nederēja. Pogas nosaukuma maiņa tobrīd vairs nešķita aktuāla.